doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -436,34 +447,68 @@ is_midplane_reachable(self):
 #### 3.1.5 ModuleBase class new APIs
 
 ##### 3.1.5.1 Need for consistent storage and access of DPU reboot cause, state and health
-1.  The smartswitch needs to know the reboot cause for DPUs. Please refer to the CLI section for the various options and their effects when executed on the switch and DPUs. 
-
-* Each DPU will update its reboot cause history in the Switch ChasissStateDB upon boot up. The recent reboot-cause can be derived from that list of reboot-causes.
-* The get_reboot_cause will return the current reboot-cause of the module.
-* For persistent storage of the DPU reboot-cause and reboot-caue-history files use the existing host storage path and mechanism.
-
-#### Schema for REBOOT_CAUSE - switch stateDB
+#### Reboot Cause
+1. The smartswitch needs to know the reboot cause for DPUs. Please refer to the CLI section for the various "show reboot-cause" options and their effects.
+    * Each DPU will update its reboot cause history in the Switch ChassisStateDB upon boot up and also persist this on the host. The recent reboot-cause is derived from that list of reboot-causes.
```

Comment:
@rameshraghupathy it seems some agent in the DPU is responsible for pushing the reboot cause...so this will not work if the midplane is down for whatever reason.

Reply:
> @rameshraghupathy it seems some agent in the DPU is responsible for pushing the reboot cause...so this will not work if the midplane is down for whatever reason.

@prgeor  1. The switch hardware is capable of getting the DPU reboot-cause on the NPU side and the required software work is in progress to provide it when requested with "get_reboot_cause()" even if the DPU is not reachable though the midplane.  2. We will have support to indicate a DPU reboot every time a DPU reboots.  The existing workflow will remain the same but how the platform code fetches the reboot-cause will be as explained above when the implementation is ready.